scripts/mirror-builtin-actors.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# This script mirrors all releases of Filecoin's builtin-actors that have been updated in the past two years,
+# It respects GitHub API rate limits and paginates requests.
+# It performs the following operations:
+# - Downloads release assets from GitHub.
+# - Compares these assets with the existing ones in an S3 bucket.
+# - Uploads new or updated assets to the S3 bucket.
+# - Sends an alert to Slack if any uploads fail.
+
+set -eo pipefail
+
+export DEBIAN_FRONTEND=noninteractive
+
+BASE_FOLDER="$(pwd)/releases/actors"
+API_URL="https://api.github.com/repos/filecoin-project/builtin-actors/releases"
+FAILED_LOG="$(pwd)/failed_uploads.log"
```

Comment:
This file is never read, and it is no longer available after the action is complete. What's its purpose?

Reply:
pushed the wrong last commit, it's read now